cmd/api/README.md

```diff
@@ -11,6 +11,7 @@
 | DB_NAME              | MySQL Database name             | tekton_results                               |
 | DB_SSLMODE           | Database SSL mode               | verify-full                                  |
 
+These values can also be set in the config file located in the [env](https://github.com/tektoncd/results/env/config) directory. 
```

Comment:
This is creating a broken link, causing the build test to fail. The link probably can't be added until this PR merges.

```

----------------------------------------------
---- Checking links in the markdown files ----
----------------------------------------------
cmd/api/README.md
	ERROR	https://github.com/tektoncd/results/env/config
		Not Found (HTTP error 404)
```

Reply:
Remove the url and replaced with folder path. I will update the README with the correct link in another PR after this has been merged.